### DISCUSSION TITLE
[求指点]我这个号还能要吗？

### DISCUSSION THREAD

**Discussion by L502567** [share121/inter-knot#2078] [其他]
![Screenshot_2024-08-14-15-10-51-97_24b09ab753950a7e51b9b33271c35069](https://github.com/user-attachments/assets/f561d3a7-6e24-41c4-a4df-01e4b5851648)

**Comment by fufumo** [comment]
牛逼

**Comment by hfwang132** [comment]
确实废了，抽到鲨鱼妹没有狼哥，抽青衣的话又没有朱鸢。火队开服也没什么加成，
